### PR TITLE
fix: resolve issue where verify validation did not properly validate uniqueness across all modules

### LINKED ISSUE #8372: skaffold verify "no duplicate names" validation currently doesn't properly handle multiple modules
Issue tracks known bug where skaffold verify "no duplicate names" validation currently doesn't properly handle multiple modules

### PR DESCRIPTION
**Pull request by aaron-prindle**
fixes #8372

### CODE REVIEW

File: pkg/skaffold/schema/validation/validation.go [hunk 1]
```diff
@@ -726,30 +726,25 @@ func validateLogPrefix(cfg *parser.SkaffoldConfigEntry, lc latest.LogsConfig) []
...
+	}
+	for _, tc := range tcs {
 		if _, ok := seenTestName[tc.Name]; ok {
-			return []ErrorWithLocation{
-				{
-					Error:    fmt.Errorf("found duplicate test name '%s' in 'verify' test cases. 'verify' test case names must be unique", tc.Name),
-					Location: cfg.YAMLInfos.Locate(&cfg.Verify[i]),
-				},
-			}
+			errs = append(errs, fmt.Errorf("found duplicate test name '%s' in 'verify' test cases. 'verify' test case names must be unique", tc.Name))
```
Comment (gsquared94) [inline pkg/skaffold/schema/validation/validation.go:739]: is it possible to get the file name if these verify stanzas are from different `skaffold.yaml` files?

Comment (aaron-prindle) [reply pkg/skaffold/schema/validation/validation.go:739]: I don't believe this information is available in `validation.go` for `skaffold.yaml` files.  I don't think we have the rel/abs path of the SkaffoldConfig(s) stored in any objects we pass around.  Only the flag runCtx.ConfiguraionFile() I believe has something close.  Are you familiar w/ a place this is stored?

**Comment by aaron-prindle** [comment]
I don't think it's possible to add the skaffold.yaml files names with the current values we track.  Lmk if there is any other changes, etc. to made here or if you think it is possible.  Thanks!

**Review by gsquared94 — APPROVED** [review]
LGTM.

**Pull request merged**